### PR TITLE
[SIG-2558] Add 'wonen' subcategory

### PR DESCRIPTION
**Pull request by jpoppe**
This pull request also implements the following stories: SIG-2592, SIG-2582, SIG-2583, SIG-2584, SIG-2585, SIG-2031

After merging this branch: SIG-2031 could also be removed

### CODE REVIEW

File: src/signals/incident/containers/IncidentContainer/debug/initialState.js [hunk 1]
```diff
@@ -20,10 +20,10 @@ export default {
       ],
     },
   },
-  category: 'overlast-bedrijven-en-horeca',
-  subcategory: 'stankoverlast',
+  category: 'wonen',
```
Comment (aditudorache) [inline src/signals/incident/containers/IncidentContainer/debug/initialState.js:23]: is this file used anyware? I think  or can it be removed.

File: src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js [hunk 2]
```diff
@@ -0,0 +1,864 @@
...
+            'woningdelen',
+          ],
+        },
+        label: 'Weet u wat zich in deze woning afspeelt?',
+        subtitle: 'Vermoedens over bijvoorbeeld illegale activiteiten',
+        shortLabel: 'Vermoeden',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+      },
+      render: FormComponents.TextInput,
```
Comment (janjaap) [inline src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js:32]: Shouldn't this be a textarea?

File: src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js [hunk 3]
```diff
@@ -0,0 +1,864 @@
...
+            'woningdelen',
+          ],
+        },
+        label: 'Weet u wie de eigenaar is van de woning?',
+        shortLabel: 'Naam Eigenaar',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+      },
+      options: {
+        validators: [Validators.required],
```
Comment (janjaap) [inline src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js:47]: Should this be required? If so, shouldn't it be a set of radio buttons?

File: src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js [hunk 4]
```diff
@@ -0,0 +1,864 @@
...
+          subcategory: 'onderhuur',
+        },
+        label: 'Wat zijn de namen van de mensen die op dit adres wonen?',
+        shortLabel: 'Naam bewoners',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+        values: {
+          ja: 'Ja, de bewoners zijn familie',
+          nee: 'Nee, de bewoners zijn geen familie',
+          weet_ik_niet: 'Weet ik niet',
```
Comment (janjaap) [inline src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js:239]: This doesn't seem to be correct

File: src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js [hunk 5]
```diff
@@ -0,0 +1,864 @@
...
+          ],
+        },
+        label: 'Weet u wie de officiële huurder is van de woning?',
+        subtitle: 'De persoon die in de woning zou moeten wonen',
+        shortLabel: 'Naam huurder',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+      },
+      options: {
+        validators: [Validators.required],
```
Comment (janjaap) [inline src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js:280]: Should this be required? Answer to the question can be yes or no.

File: src/signals/incident/containers/IncidentContainer/debug/initialState.js (hunk 1 above)
Comment (jpoppe) [reply src/signals/incident/containers/IncidentContainer/debug/initialState.js:23]: Was thinking about this, but it was already there and not create by me so :man_shrugging:

Comment (jpoppe) [reply src/signals/incident/containers/IncidentContainer/debug/initialState.js:23]: @janjaap could you shine a light on this? Thanks!

File: src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js (hunk 2 above)
Comment (jpoppe) [reply src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js:32]: Herjen says no.

File: src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js (hunk 5 above)
Comment (jpoppe) [reply src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js:275]: Herjen wants to keep it like it is.

File: src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js (hunk 3 above)
Comment (jpoppe) [reply src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js:47]: Herjen would like to keep this as it is.

File: src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js (hunk 4 above)
Comment (jpoppe) [reply src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js]: Fixed!

File: src/signals/incident/containers/IncidentContainer/debug/initialState.js (hunk 1 above)
Comment (janjaap) [reply src/signals/incident/containers/IncidentContainer/debug/initialState.js:23]: This file is there, I believe, to help trigger a particular category and subcategory's set of questions. I'd say remove this file.

Comment (jpoppe) [reply src/signals/incident/containers/IncidentContainer/debug/initialState.js:23]: Thanks!

**Pull request merged**